application-analytics-default/src/main/java/com/xwiki/analytics/script/AnalyticsScriptService.java

```diff
@@ -75,11 +90,19 @@ public JsonNode getMatomoRequestResult(Map<String, String> parameters, Map<Strin
     /**
      *
      * @return the analytics configuration.
-     * @since 1.0 I must add the @since to be able to build the application
      */
-    @Unstable
     public AnalyticsConfiguration getConfiguration()
     {
         return configuration;
     }
+
+    private AnalyticsManager getManager(String hint)
+    {
+        for (AnalyticsManager manager : managerProvider.get()) {
+            if (hint.equals(manager.getIdentifier())) {
```

Comment:
In the future, when we'll have multiple implementations, we'll use the context Component Manager to get the ``AnalyticsManager`` instance with the specific hint:

    return this.contextComponentManager.getInstance(AnalyticsManager.class, hint);

But we don't need this now.